### PR TITLE
Fix docstrings for TF BLIP

### PR DESCRIPTION
**Pull request by Rocketknight1**
Some of the docstrings were still a bit PyTorchy, this is fixed now! (cc @ydshieh)

### CODE REVIEW

**Comment by innat** [comment]
@Rocketknight1 Thanks for adding tf-blip. 
Regarding documentaiton, the blip-captiong model is configured to cal. loss inside the model 

```python
outputs = model(input_ids=input_ids, pixel_values=pixel_values, labels=input_ids)
loss = outputs.loss
```
Would it be possible to follow keras-way, passing loss via model.compile? 

PS. in case you're interested to contribute https://github.com/keras-team/keras-nlp/issues/941

**Comment by ydshieh** [comment]
@Rocketknight1  I tried to re-run the CI, but it still fails. Could you push an empty commit to trigger it maybe?

**Comment by Rocketknight1** [comment]
@ydshieh tests are passing now! Can you approve the PR?

**Comment by Rocketknight1** [comment]
@ydshieh tests should pass now! The cause was some expected values in the tests being wrong. I copied the right ones from the torch tests and now everything is passing locally, so hopefully the CI will agree.

**Comment by Rocketknight1** [comment]
The doctests are weird, though - I think some of them were broken in PyTorch too. Working on it!

**Comment by ydshieh** [comment]
Thanks @Rocketknight1 I will double check. But do you figure out (some of) tests pass on CircleCI but fails on daily CI - the pt<->tf equivalence tests also run on CircleCI. We should see they fail on it (as they fail on daily CI).

**Comment by ydshieh** [comment]
For the doctest, we still get 

```python
Expected:
    two cats are laying on a couch
Got:
    two cats sleeping on a couch
```
for `TFBlipForConditionalGeneration.generate`.

Regarding the modeling tests - all tests pass now 🥳

**Review by ydshieh — COMMENTED** [review]
LGTM, thanks @Rocketknight1 on this. A doctest value still needs to be updated I think.

Also wondering why we don't see test failure on CircleCI previously. If you know, please let me know 🙏 .

**Comment by Rocketknight1** [comment]
@ydshieh should be resolved now!

**Review by ydshieh — APPROVED** [review]
Thank you @Rocketknight1 again, our 2 cute cats can finally sleep well now.
